README.md

```diff
@@ -1,9 +1,24 @@
 ![logo](https://raw.githubusercontent.com/Kesomannen/gale/master/app-icon@0,25x.png)
 
-# Gale Mod Manager
+<h2> Gale Mod Manager </h2>
```

Comment:
Why change this to h2? Isn't that just the same as "##"?

Reply:
It _is_ the same, I used a tool to both build and update the table of contents and it will add anything with regular markdown hashes into the ToC. Its a VS extension called Markdown All In One and I highly recommend it

Reply:
Ah ok, I think it should be <h1> though if that works

Reply:
I have embiggened the title :3